src/content-scripts/ActionMgr.js

```diff
@@ -112,12 +108,12 @@ class ActionMgr {
 					new Set(
 						SelectedElements
 							.Links
-							.map(elem => elem.getAttribute('href'))
+							.map(elem => elem.href)
 					)
 				);
 
-				if(e.ctrlKey)
-					this.CopyToClipboard(links);
+				if ((e.ctrlKey && Prefs.DefaultAction == '1') || (!(e.ctrlKey) && Prefs.DefaultAction == '2'))
```

Comment:
I'd prefer to use text values, such as Prefs.DefaultAction == 'OpenLinks'.  Failing that they should be constants, such as:

```js
// globals.js
const OPENLINKS = 1
const COPY_LINKS = 2
```

Either way works and it keeps the code clean and clear.

Reply:
I modified the code to use costants